.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
**Issue:**

This is not required in your commit. 

**Resolution:**

Please avoid staging changes not essential to the amendments you have made.